lib/store_model/model.rb

```diff
@@ -110,5 +113,16 @@ def _has_attribute?(attr_name)
     def unknown_attributes
       @unknown_attributes ||= {}
     end
+
+    private
+
+    def attribute?(attribute)
+      case value = attributes[attribute]
+      when true then true
+      when false, nil, 0 then false
+      else
+        value.present?
+      end
```

Comment:
looks like we can reduce it to
```suggestion
      when 0 then false
      else value.present?
      end
```